fe/fe-core/src/test/java/com/starrocks/sql/plan/ExternalTableTest.java

```diff
@@ -160,7 +186,7 @@ public void testJDBCTableFilter() throws Exception {
         String plan = getFragmentPlan(sql);
         Assert.assertTrue(plan, plan.contains("0:SCAN JDBC\n" +
                 "     TABLE: `test_table`\n" +
-                "     QUERY: SELECT a, b, c FROM `test_table` WHERE (a > 10) AND (b < 'abc')\n" +
+                "     QUERY: SELECT a, b, c FROM `test_table` WHERE (`a` > 10) AND (`b` < 'abc')\n" +
```

Comment:
This is not a good case, you should change it to 
```suggestion
                "     QUERY: SELECT `a`, `b`, c FROM `test_table` WHERE (`a` > 10) AND (`b` < 'abc')\n" +
```
Because `a`, `b` must be a keyword in the select list.

Reply:
done, move all reserved key word logic to the same place.